content/en/docs/collector/deployment.md

```diff
@@ -1,32 +1,181 @@
 weight: 2
 ---
 
-The OpenTelemetry Collector consists of a single binary and two primary deployment methods:
-
-- **Agent:** A Collector instance running with the application or on the same
-  host as the application (e.g. binary, sidecar, or daemonset).
-- **Gateway:** One or more Collector instances running as a standalone service
-  (e.g. container or deployment) typically per cluster, data center or region.
-
-## Agent
-
-It is recommended to deploy the Agent on every host within an environment. In
-doing so, the Agent is capable of receiving telemetry data (push and pull
-based) as well as enhancing telemetry data with metadata such as custom tags or
-infrastructure information. In addition, the Agent can offload responsibilities
-that client instrumentation would otherwise need to handle including batching,
-retry, encryption, compression and more.
-
-## Gateway
-
-Additionally, a Gateway cluster can be deployed in every cluster, data center,
-or region. A Gateway cluster runs as a standalone service and can offer
-advanced capabilities over the Agent including tail-based sampling. In
-addition, a Gateway cluster can limit the number of egress points required to
-send data as well as consolidate API token management. Each Collector instance
-in a Gateway cluster operates independently so it is easy to scale the
-architecture based on performance needs with a simple load balancer. If a
-gateway cluster is deployed, it usually receives data from Agents deployed
-within an environment.
+The OpenTelemetry collector consists of a single binary which you can use in
+different ways, for different use cases. This document describes deployment
+patterns, their use cases along with pros and cons.
+
+## Decentralized Deployment
+
+The decentralized collector deployment pattern consists of applications,
+[instrumented][instrumentation] with an OpenTelemetry SDK, using [OpenTelemetry
+protocol (OTLP)][otlp] to send telemetry data to one or more
+[collectors][collector], with each client-side SDK configured with a collector
+location:
+
+![OpAMP example setup](../img/decentralized-sdk.svg)
+
+1. In the app, the SDK is configured to send OTLP data to a collector.
+1. The collector is configured to send telemetry data to a backend.
+
+### Example
+
+A concrete example of the decentralized collector deployment pattern could look
+as follows: you manually instrument, say, a [Java application to export
+metrics][instrument-java-metrics] using the OpenTelemetry Java SDK. In the
+context of the app, you would set the `OTEL_METRICS_EXPORTER` to `otlp` (which
+is the default value) and configure the [OTLP exporter][otlp-exporter] with the
+address of your collector, for example (in Bash or `zsh` shell):
+
+```
+export OTEL_EXPORTER_OTLP_ENDPOINT=http://collector.example.com:4318
+```
+
+The collector would then be configured like so:
+
+{{< ot-tabs Traces Metrics Logs >}} {{< ot-tab lang="yaml">}}
```

Comment:
FYI, we're trying to phase out the use of the `ot-tabs` shortcode (for details see #1820). Since you seem to be on a roll in terms of creating new content, would you be willing to switch to using the [**Docsy tabpane shortcode**](https://www.docsy.dev/docs/adding-content/shortcodes/#tabbed-panes) instead?

Here's an example: https://github.com/open-telemetry/opentelemetry.io/blob/e3edb6217221c7301313278203d8579a9b75b5c2/content/en/docs/instrumentation/js/instrumentation.md?plain=1#L39-L105

Reply:
OK, I got it to work after some wrangling, here's what I'm using:

```
<!-- prettier-ignore-start -->
{{< tabpane persistLang=false >}}
  {{< tab header="Traces" lang="yaml" >}}
...
```